components/collapse/demo/accordion.ts

```diff
@@ -4,9 +4,11 @@ import { Component } from '@angular/core';
   selector: 'nz-demo-collapse-accordion',
   template: `
     <nz-collapse nzAccordion>
-      <nz-collapse-panel *ngFor="let panel of panels" [nzHeader]="panel.name" [nzActive]="panel.active">
-        <p>{{ panel.name }} content</p>
-      </nz-collapse-panel>
+      @for (panel of panels; track panel.name) {
```

Comment:
We cannot ensure that panel.name is unique.

Reply:
At this moment $index is more appropriate or just panel

Reply:
Let's use `panel` :)

Reply:
Will change this today